### PR TITLE
Task 3: Item Pop-up menu

### PR DESCRIPTION
**Pull request by AsumVictor**
# Task 3:  Item Pop-up menu
In this PR, I have added functionality to display a popup menu and display details of the movie selected
 
 ## Type of change
 * [ ] Created a new file called `modalBox.js`
 * [ ] Created function in the modalbox.js to display the popup menu
 * [ ] Updated the displayMovie function: Added event listener for comment button for each card
 * [ ] Updated the style.css file: Added the basic style of the popup-me
 
 # Checklist:
 * [ ]  My code follows the style guidelines of this project
 * [ ] I have performed a self-review of my code
 * [ ] There is no linter error in my code
 * [ ]  My changes generate no new warnings

### CODE REVIEW

**Review by Dorcas126 — CHANGES_REQUESTED** [review]
Hi @AsumVictor Good job working on the popup window👌
AlonzoLeroneThisCloseGIF

kindly, add a clear description of what you did,
There are linters errors too.

Kindly, work on those changes.
keep it up👍

File: src/template.html [hunk 1]
```diff
@@ -42,9 +42,25 @@ <h3 class="movies-length"></h3>
...
+           <div class="info">
+             <p>Likes: 453</p>
+             <p>Type: Romans, Love</p>
+             <p>Language: english</p>
+             <p>Official Site: visit site</p>
+           </div>
+           <p class="comments-head">Comments</p>
+           <hr>
+         </div>
+      </div> -->
```
Comment (Dorcas126) [inline src/template.html]: Hi Victor, Kindly remove the commented-out code as it makes the code looks unclean.

Comment (AsumVictor) [reply src/template.html]: Great! I will remove it in my next commit ASAP.

**Review by Dorcas126 — APPROVED** [review]
Hi @AsumVictor , congratulations🍾
It's time to merge your project
well done!

**Comment by AsumVictor** [comment]
> Hi @AsumVictor , congratulations🍾 It's time to merge your project well done!

Great!

**Pull request merged**